js/forms.js

```diff
@@ -13,12 +13,14 @@ var State =  require('./State');
 var FileInput = require('./FileInput');
 var Select = require('./Select');
 
-var Forms = function(selectors) {
+var Forms = {};
+
+Forms.init = function(selectors) {
 	selectors = selectors || {};
 
 	FileInput(selectors.fileInput);
 	Select(selectors.select);
-	State(selectors.state);
+	State.init();
```

Comment:
It is possible to change the selectors for either the FileInput, Select and State. So for instance, in this implementation,

Forms.init({ fileInput: '.someSelector', select: '.someSelector2', state: '.someSelector3' });

These value need to be passed through to the modules accordingly.